PWGGA/PHOSTasks/PHOS_gamma/AddTaskGammaPHOSPP.C

```diff
@@ -22,6 +27,20 @@ AliAnalysisTaskGammaPHOSPP* AddTaskGammaPHOSPP( Bool_t isMC,
     if (!mgr->GetInputEventHandler()) {
         return 0x0;
     }
+
+    //Tender & response
+    
+    const Double_t zsSimulation = isMC ? 0.020 : 0;
+ 
+    AliPHOSTenderTask *tenderPHOS = reinterpret_cast<AliPHOSTenderTask *>
+        (gInterpreter->ExecuteMacro(
+         Form("$ALICE_PHYSICS/PWGGA/PHOSTasks/PHOS_PbPb/AddAODPHOSTender.C(\"%s\", \"%s\", \"%s\", %d, %d, \"%s\", %f)", 
+                           "PHOSTenderTask","PHOStender",tenderOption.Data(), tenderPass, isMC, nonlinearity.Data(), zsSimulation)));
```

Comment:
Normally tasks like Tender or PIDresponse are called in independent wagons or config macros. Your approach will create another Tender in case if one Tender was already created in special wagon or previous instance of your task and this will result in definitely wrong results. Why do you call these wagons within your analysis task? At least, put check that Tender was not already created, but better move external wagons to confg. macros.

Reply:
Hello Dmitri,

Thank you for your comment.

I will remove these lines in my new commit